src/components/Books.js

```diff
@@ -0,0 +1,11 @@
+import Book from './Book';
+
+const Books = () => (
+  <>
+    <Book />
+    <Book />
+    <Book />
```

Comment:
- [ ] Please takes the book data as a prop and render the `title` and the `author` check below :point_down: my suggestion :slightly_smiling_face: 

```suggestion
      <Book title="The Pilgrim's Progress" author="John Bunyan" />
      <Book title="Robinson Crusoe" author="Daniel Defoe" />
```